src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -375,6 +375,43 @@ def get_validate_platform(cmd, os_type, platform):
     return platform_os, platform_arch, platform_variant
 
 
+def get_yaml_and_values(cmd_value, timeout, file):
+    """Generates yaml template and its value content if applicable
+    :param str cmd_value: The command to execute in each step
+    :param str timeout: The timeout for each step
+    :param str file: The task definition
+    """
+    yaml_template = ""
+    values_content = ""
+    if cmd_value:
+        yaml_template = "steps: \n  - cmd: {{ .Values.command }}\n"
+        values_content = "command: {0}\n".format(cmd_value)
+        if timeout:
+            yaml_template += "    timeout: {{ .Values.timeout }}\n"
+            values_content += "timeout: {0}\n".format(timeout)
+    else:
+        if not file:
+            file = "acb.yml"
```

Comment:
Previously the default file name passed to `acr_run` is in `acb.yaml`. Can you change back to it?